src/app/parcels/parcelsTable/filters.ts

```diff
@@ -119,7 +119,10 @@ const buildLastStatusFilter = async (): Promise<ParcelsFilter<string[]>> => {
     const lastStatusSearch: ParcelsFilterMethod<string[]> = (query, state) => {
         if (state.length === 0) {
             // Default is to show everything that's not deleted
-            return query.neq("last_status_event_name", "Parcel Deleted");
+            return query.or(
+                // eslint-disable-next-line quotes
```

Comment:
VSCode's lint rules didn't seem to be happy with any combination of quotes, and below is readable